_test/test_gen_sqw_workflow/test_gen_sqw_accumulate_sqw_herbert.m

```diff
@@ -176,7 +176,7 @@ function test_worker_operations_and_communications(obj)
             end
             assertElementsAlmostEqual(res.data_range,data_range);
             
-            assertEqual(numel(det.group),96);
+            %assertEqual(numel(det.group),96);
```

Comment:
it would be nice to resume testing for number of detectors, but it is not immediately obvious how to do it so leaving this line with additional comment to that effect to remind me to do it